File: src/main/java/com/raoulvdberge/refinedstorage/api/storage/tracker/StorageTrackerEntry.java
package com.raoulvdberge.refinedstorage.api.storage.tracker;

/**
 * Represents a storage tracker entry.
 */
public class StorageTrackerEntry {
    private final long time;
    private final String name;

    public StorageTrackerEntry(long time, String name) {
        this.time = time;
        this.name = name;
    }

    /**
     * @return the modification time
     */
    public long getTime() {
        return time;
    }

    /**
     * @return the name of the player who modified this item
     */
    public String getName() {
        return name;
    }
}

File: src/main/kotlin/com/refinedmods/refinedstorage/api/storage/tracker/StorageTrackerEntry.kt
package com.refinedmods.refinedstorage.api.storage.tracker



/**
 * Represents a storage tracker entry.
 */
class StorageTrackerEntry(private val time: Long, private val name: String) {
    /**
     * @return the modification time
     */
    fun getTime(): Long {
        return time
    }

    /**
     * @return the name of the player who modified this item
     */
    fun getName(): String {
        return name
    }
}